### DISCUSSION TITLE
Connect a domain who is not on AWS

### DISCUSSION THREAD

**Discussion by MyVersep** [mozilla/hubs#5606] [Hubs Cloud Questions] [answered]
I need to create a hubs by mozilla cloud, for this I need to register a domain on AWS first on ROUTE53
https://discord.com/channels/498741086295031808/819188464145924117/999963748192235550

However my domain is on another host, if I transfer it I will lose all my data (mailing, files), I would like to be able to do it without transferring but I don't know how, I saw that it was necessary to create a hosted zone but I don't think that Hubs by mozilla takes it into account.

If anyone has an idea?

**Comment by pattersonbl2** [comment, marked as answer]
You would use domain recipe 3.      https://hubs.mozilla.com/docs/hubs-cloud-aws-domain-recipes.html
